Clean Swift/Scene.xctemplate/UIViewController/___FILEBASENAME___Worker.swift

```diff
@@ -17,14 +17,22 @@ class ___VARIABLE_sceneName___Worker {
 
     // MARK: - Methods
 
+    // MARK: Fetch From Remote DataStore
+
+    func fetchFromRemoteDataStore() -> String {
```

Comment:
@zaimramlan since this is fetch from remote, can it have completion handler instead of return?

Reply:
Same reply as above.